client/allocrunner/interfaces/runner_lifecycle.go

```diff
@@ -53,3 +53,10 @@ type ShutdownHook interface {
 
 	Shutdown()
 }
+
+//
+type RunnerPreKillHook interface {
```

Comment:
Lets drop the `Runner` prefix here and follow the format of the other Hook interfaces.

Reply:
ShutdownHook is the only one without the `Runner` prefix which is used by both AllocRunner and TaskRunner I believe. Does it make sense to keep the Runner prefix in the interface name and just have its method be `PreKillHook`?

Reply:
Oh ha you're right disregard!